src/NetAnalyzers/Core/Microsoft.CodeQuality.Analyzers/QualityGuidelines/DoNotInitializeUnnecessarily.Fixer.cs

```diff
@@ -0,0 +1,68 @@
+// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using System.Threading;
+using System.Threading.Tasks;
+using Analyzer.Utilities;
+using Microsoft.CodeAnalysis;
+using Microsoft.CodeAnalysis.CodeFixes;
+using Microsoft.CodeAnalysis.CSharp.Syntax;
```

Comment:
Given the fixer is now C# specific and uses C# syntax APIs, we should move it to the C# layer: https://github.com/dotnet/roslyn-analyzers/tree/0fcf66bca3d6748be1fa4df89b4778a79c90b080/src/NetAnalyzers/CSharp